source/manage/client-annotations.rst

```diff
@@ -1,74 +1,59 @@
 .. _pmm-admin.annotate:
 
---------------------------------------------------------------------------------
+#######################################
 Annotating important Application Events
---------------------------------------------------------------------------------
+#######################################
 
-.. _pmm-admin.annotate.adding:
+******************
+Adding annotations
+******************
 
-`Adding annotations <client-annotations.html#adding-annotations>`_
---------------------------------------------------------------------------------
+The ``pmm-admin annotate`` command registers a moment in time, marking it with a text string called an *annotation*.
 
-Use the |pmm-admin.annotate| command to set notifications about important
-application events and display them on all dashboards. By using annotations, you
-can conveniently analyze the impact of application events on your database.
+The presence of an annotation shows as a vertical dashed line on a dashboard graph; the annotation text is revealed by mousing over the caret indicator below the line.
 
-.. _pmm-admin.annotate.usage:
+Annotations are useful for recording the moment of a system change or other significant application event.
 
-.. rubric:: USAGE
+They can be set globally or for specific nodes or services.
 
-|tip.run-this.root|
+.. image:: /.res/graphics/png/pmm-server.mysql-overview.mysql-client-thread-activity.1.png
 
-.. include:: .res/code/pmm-admin.annotate.tags.txt
+.. rubric:: USAGE
 
-.. _pmm-admin.annotate.options:
+``pmm-admin annotate [--node|--service] <annotation> [--tags <tags>] [--node-name=<node>] [--service-name=<service>]``
 
 .. rubric:: OPTIONS
 
-The |pmm-admin.annotate| supports the following options:
-
-|opt.tags|
-
-   Specify one or more tags applicable to the annotation that you are
-   creating. Enclose your tags in quotes and separate individual tags by a
-   comma, such as "tag 1,tag 2".
-
-You can also use
-:ref:`global options that apply to any other command <pmm-admin.options>`.
-
-.. _pmm.metrics-monitor.annotation.application-event.marking:
+``<annotation>``
+    The annotation string. If it contains spaces, it should be quoted.
 
-`Marking Important Events with Annotations <client-annotations.html#application-event-marking>`_
-------------------------------------------------------------------------------------------------------------
+``--node``
+   Annotate the current node or that specified by ``--node-name``.
 
-Some events in your application may impact your database. Annotations
-visualize these events on each dashboard of |pmm-server|.
+``--service``
+   Annotate the service running on the current node. If there is more than one service, annotate that specified by ``--service-name``.
```

Comment:
Annotate all services running on the current node. I am not sure if it is clear without word all. Also if there is more services on current node and also you define --service-name then only service with that name will be annotated.

Reply:
Agreed.